WordPressKit/BlogServiceRemote.h

```diff
@@ -22,6 +22,17 @@ typedef void (^SuccessHandler)(void);
 - (void)getAuthorsWithSuccess:(UsersHandler)success
                       failure:(void (^)(NSError *error))failure;
 
+/**
+ Synchronizes all blog's authors.
+
+ @param number Limit the total number of users returned.
+ @param success The block that will be executed on success.  Can be nil.
+ @param failure The block that will be executed on failure.  Can be nil.
+ */
+- (void)getAllAuthorsWithNumber:(NSNumber *)number
```

Comment:
I'm curious.  The `number` parameter is letting the caller define the batch size, but is there a need?  Would it make more sense to just have `getAllAuthorsWithSuccess:failures:` and define the batch size internally?  Just a thought.